src/ripple/app/hook/impl/applyHook.cpp

```diff
@@ -2316,25 +2316,41 @@ DEFINE_JS_FUNCTION(
 {
     JS_HOOK_SETUP();
 
-    auto val =
-        FromJSIntArrayOrHexString(ctx, raw_val, hook::maxHookStateDataSize());
-    auto key_in = FromJSIntArrayOrHexString(ctx, raw_key, 32);
-    auto ns_in = FromJSIntArrayOrHexString(ctx, raw_ns, 32);
-    auto acc_in = FromJSIntArrayOrHexString(ctx, raw_acc, 20);
+    auto val = FromJSIntArrayOrHexString(ctx, raw_val, 0x10000);
```

Comment:
This doesn't make sense, FromJSIntArrayOrHexString will reserve 65,536 bytes for each of these: 

https://github.com/Xahau/xahaud/blob/ab529a4a1035843411e1a53e5fb2a7b950d091ed/src/ripple/app/hook/impl/applyHook.cpp#L1006-L1013